### ISSUE TITLE
Keri RFID NXT not reading

### ISSUE THREAD

**Issue by devopgirlie** [flipperdevices/flipperzero-firmware#3494]
Keri key fob NXT series (starts with N before serial number) does not read under RFID or NFC. Unable to get a RAW read as well. Is there a fix to this issue? Thanks in advance

**Comment by skotopes** [comment]
@devopgirlie where can we find those fobs? are they on sale anywhere?

@doomwastaken can we get some?

**Comment by doomwastaken** [comment]
Yeah, I found them for another ticket a while back, will order them

**Comment by devopgirlie** [comment]
@skotopes @doomwastaken found them here https://www.idzone.com/nxt-k-pf.html

here are the specs from the site: https://kerisys.com/products/nxt-credentials/

**Comment by doomwastaken** [comment]
The one I had in mind is https://www.dkhardware.com/nxt-k-keri-systems-proxfob-nxt-k-product-1138628.html

**Comment by amanbaissa** [comment]
@doomwastaken has there been an update on reading NXT keri cards?

**Comment by Delso411** [comment]
It has been no update to my knowledge i always try once in a while to copy keri nxt series key fob which is my main keyfob and i had lost one and been trying to copy one for the longest

**Comment by doomwastaken** [comment]
We had recently ordered key fobs, sadly I was unable to find them in the UK. Once they arrive, we will get back to you. Sorry for delay

**Comment by skotopes** [comment]
@devopgirlie just arrived, we'll take a look on this week

**Comment by 0xDegenDeveloper** [comment]
Looking forward to an update on this, been trying to copy my sauna's fob for 2+ years now. Its a Keri NX... tag, with all the firmwares I've had no luck. 

Curious if we expect the NX tags to be possible with a flipper, or if I should look into other devices to do so.

**Comment by skotopes** [comment]
Those tags are indeed quite unusual. We'll need more time to add their support.

**Comment by OnlineMartialArtist** [comment]
I couldn't get the flipper to read my Keri RFID card, but Keysey was able to read it and clone it. Keysey is a small device for copying/cloning/erasing an RFID card and it's sold on Amazon. To this day, I can't get the flipper to read my Keri fob. 🤷‍♂️

**Comment by forxhunter** [comment]
any progress?

**Comment by OnlineMartialArtist** [comment]
No progress yet, but I’ve been working on it a lot recently because someone in my building just got a flipper and we’ve been tinkering/researching constantly. I know someone made this script for raw files to RFID in python, but I haven’t tried it yet. I tried the plotter for the raw files and it didn’t give me anything I could see that was usable to create a key manually for Keri in hex (which is now an option). 

here’s the link for the python code:

https://github.com/hnesk/flipper-raw-rfid

PLEASE ADD SUPPORT FOR KERI NXT RFID! Ty!

**Comment by RetailCircle** [comment]
I couldn't read and clone my KERI NXT-K fob today either. If someone has a solution, please let me know.

**Comment by Ravioli-cpu** [comment]
Interested in support being added for this as well

**Comment by amanbaissa** [comment]
@skotopes any updates to reading the NXT fobs?

**Comment by daltonguitar** [comment]
@skotopes Any updates on the western front?

**Comment by Zuldraco** [comment]
Silly of me to ask, but any updates?

**Comment by Ravioli-cpu** [comment]
I think we gotta pack it up yall they've been radio silent for forever
